setup.py

```diff
@@ -17,58 +17,65 @@
 from os import path
 
 here = path.abspath(path.dirname(__file__))
-with open('requirements.txt') as f:
-    require = f.readlines()
-install_requires = [r.strip() for r in require if
-                    not r.startswith('http') and not r.startswith('#')]
+# with open('requirements.txt') as f:
+#     require = f.readlines()
+# install_requires = [r.strip() for r in require if
+#                     not r.startswith('http') and not r.startswith('#')]
```

Comment:
Does this properly handle the 
`-e git+https://github.com/ODM2/geoalchemy.git@odm2#egg=geoalchemy-0.7.3`